lib/components/map/default-map.tsx

```diff
@@ -462,6 +504,8 @@ class DefaultMap extends Component {
 const mapStateToProps = (state) => {
   const activeSearch = getActiveSearch(state)
   const viewedRoute = state.otp?.ui?.viewedRoute?.routeId
+  const nearbyFilters = state.otp?.ui?.nearbyView?.filters
+  const nearbyFiltersConfigured = state.otp.config.nearbyView.filterConfig
```

Comment:
These are named quite confusingly. What do you think about this?
```suggestion
  const activeNearbyFilters = state.otp?.ui?.nearbyView?.filters
  const nearbyFilters = state.otp.config.nearbyView.filters
```

Reply:
Also, can we add optionals here? This is crashing if the nearby view isn't configured